components/common/Item.tsx

```diff
@@ -0,0 +1,216 @@
+import styled from 'styled-components';
+import { calcDateDiff, dateToString } from '@/common/date';
+import Link from 'next/link';
+import Image from 'next/image';
+import noImage from '@/assets/images/noImage.jpg';
+import star from '@/assets/images/star.png';
+import kebbab from '@/assets/images/kebab.png';
+import { useEffect, useRef, useState } from 'react';
+import KebbabPopover from '@/components/folder/KebbabPopover';
+import DeleteModal from '@/components/folder/DeleteModal';
+import AddLinkModal from '@/components/folder/AddLinkModal';
+import { FetchData, FolderData } from '@/common/api';
+import { SIZE } from '@/constants/size';
+
+const StyledArticle = styled.article`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  border-radius: 10px;
+  box-shadow: 0px 0px 3px #808080;
+
+  @media screen and (min-width: ${SIZE.tablet.minWidth}) {
+    max-width: 340px;
+    height: 334px;
+  }
+
+  @media screen and (max-width: ${SIZE.mobile.maxWidth}) {
+    width: 100%;
+    height: 327px;
+  }
+
+  &:hover {
+    border: 2px solid #6d6afe;
+    margin: -2px;
+  }
+`;
+
+const Content = styled.div`
+  box-sizing: border-box;
+  height: 135px;
+  padding: 15px 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+`;
+
+const ImgDiv = styled.div`
+  width: 100%;
+  height: 200px;
+  overflow: hidden;
+`;
+
+const StyledImg = styled(Image)`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  &:hover {
+    transform: scale(1.3);
+  }
+`;
+
+const CardHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 17px;
+`;
+
+const Time = styled.span`
+  font-size: 13px;
+  color: #666666;
+`;
+
+const KebbabButton = styled.button`
+  position: relative;
+  border: none;
+  background: none;
+  cursor: pointer;
+`;
+
+const Detail = styled.span`
+  height: 49px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  font-size: 16px;
+  line-height: 24px;
+  color: black;
+`;
+
+const CreatedDate = styled.span`
+  height: 19px;
+  font-size: 14px;
+  color: #333333;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`;
+
+const KebbabDiv = styled.div``;
+
+const ModalDiv = styled.div`
+  position: relative;
+`;
+
+const StarImg = styled(Image)`
+  position: absolute;
+  width: 34px;
+  height: 34px;
+  top: 15px;
+  right: 15px;
+  z-index: 99;
+`;
+
+interface ItemProps {
+  createdAt: string;
+  url: string;
+  title: string;
+  description?: string;
+  imageSource?: string;
+  folderData?: FetchData<FolderData[]>;
+  editable?: boolean;
+}
+
+function Item({ createdAt, url, title, description, imageSource, folderData, editable }: ItemProps) {
+  const [openPopover, setOpenPopover] = useState(false);
```

Comment:
boolean 값의 변수 이름은 보통 is~, has~ 로 짓는 게 일반적이라 추천드립니다!
```suggestion
  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
```